### PR TITLE
CLDR-17921 merge DDL v47 branch to main

### PR DESCRIPTION
**Pull request by srl295**
To review:
- please go to the Commits tab and add comments on the individual PRs or tickets.

To update this branch:

```shell
$ git fetch upstream
$ git merge -m 'CLDR-17921 merge from main' upstream/main
$ git push
```

CLDR-17921

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true
DISABLE_JIRA_ISSUE_MATCH=true

### CODE REVIEW

File: common/supplemental/likelySubtags.xml
Comment (macchiati) [inline common/supplemental/likelySubtags.xml:1049]: I think Roozbeh had some comments about this one, since it would be exceedingly rare to have Azerbaijan written in Arabic script these days.

**Review by macchiati — COMMENTED** [review]
FYI: filed https://unicode-org.atlassian.net/browse/CLDR-17989

File: common/supplemental/likelySubtags.xml
Comment (srl295) [reply common/supplemental/likelySubtags.xml:1049]: @macchiati this is a branch: i'll add a comment on the PR.

**Comment by macchiati** [comment]
Thanks

File: common/supplemental/likelySubtags.xml
Comment (conradarcturus) [reply common/supplemental/likelySubtags.xml:1049]: Roozbeh's comment was about Talysch. He said az_Arab_AZ is more likely than tly_Arab_AZ.

Since there are multiple writing systems for languages used in Azerbaijan and Arabic is one of them we needed a value here. Roozbeh said az_Arab_AZ made more sense -- even if its rare.

Comment (roozbehp) [reply common/supplemental/likelySubtags.xml:1049]: The AZ-related changes LGTM.

**Comment by srl295** [comment]
any concerns with merging? maybe confirm at TC or infra?

**Review by conradarcturus — APPROVED** [review]
Thanks for merging the changes back to main! After you merge it I'll rebase my PRs in progress.

**Comment by srl295** [comment]
@macchiati any objection on merging this to main?

**Comment by srl295** [comment]
Might as well wait for 46 release at this point.

**Comment by srl295** [comment]
I'll merge once (assuming) it passes

**Pull request merged**
